cfme/tests/configure/test_workers.py

```diff
@@ -45,9 +63,66 @@ def test_restart_workers(appliance):
              message="Wait for all original workers are back online")
 
 
+v510 = {
+    "1.1.gigabytes": 1178599424,
+    1178599424: "1.1.gigabytes",
+    "1.2.gigabytes": 1283457024,
+    1283457024: "1.2.gigabytes"
+}
+
+
+@pytest.fixture(scope="module")
+def _view(appliance):
+    return navigate_to(appliance.server, 'Workers')
+
+
+def set_memory_threshold_in_ui(appliance, worker, new_threshold, _view):
+    _view.browser.refresh()
+    mem_threshold = getattr(_view.workers, worker.dropdown)
+    if appliance.version < '5.11' and new_threshold in v510:
+        new_threshold = v510[new_threshold]
+    mem_threshold.select_by_visible_text(new_threshold.replace('.gigabytes', ' GB'))
+    _view.workers.save.click()
+    return new_threshold
+
+
+def get_memory_threshold_in_advanced_settings(appliance, worker):
+    worker_base = appliance.server.advanced_settings
+    loc = worker_base
+    steps = (['workers', 'worker_base'] +
+             [step if step != '*' else worker.advanced for step in worker.path])
+    for step in steps:
+        worker_base = loc
+        loc = loc.get(step)
+    return loc.get('memory_threshold', worker_base['defaults']['memory_threshold'])
+
+
+def set_memory_threshold_in_advanced_settings(appliance, worker, new_threshold, view):
```

Comment:
`view` is unused in this function.

Reply:
changed its name to unused and added comment :)